app-modules/case/database/factories/CaseUpdateFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Assist\Case\Database\Factories;
+
+use Assist\Case\Models\CaseItem;
+use Assist\Case\Models\CaseUpdate;
+use Assist\Case\Enums\CaseUpdateDirection;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<CaseUpdate>
+ */
+class CaseUpdateFactory extends Factory
+{
+    public function definition(): array
+    {
+        return [
+            'case_id' => CaseItem::factory(),
+            'update' => $this->faker->sentence(),
+            'internal' => $this->faker->boolean(),
+            'direction' => $this->faker->randomElement(CaseUpdateDirection::cases())->value,
```

Comment:
Does there need to be any correlation between `internal` and `direction`? I might be a bit confused on the terminology, but would an `internal` update also always be "inbound"?

Reply:
That's a good point. Theoretically, I guess `internal` is neither inbound nor outbound. But inbound does make the most sense.